opencensus/src/main/java/com/example/opencensus/Quickstart.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.opencensus;
+
+// [START monitoring_opencensus_metrics_quickstart]
+
+import com.google.common.collect.Lists;
+
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
+import io.opencensus.stats.Aggregation;
+import io.opencensus.stats.BucketBoundaries;
+import io.opencensus.stats.Measure.MeasureLong;
+import io.opencensus.stats.Stats;
+import io.opencensus.stats.StatsRecorder;
+import io.opencensus.stats.View;
+import io.opencensus.stats.View.Name;
+import io.opencensus.stats.ViewManager;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Random;
+
+public class Quickstart {
+  private static final int exportInterval = 60;
+  private static final MeasureLong latencyMs = MeasureLong.create(
+      "task_latency",
+      "The task latency in milliseconds",
+      "ms");
+  // Latency in buckets:
+  // [>=0ms, >=100ms, >=200ms, >=400ms, >=1s, >=2s, >=4s]
+  private static final BucketBoundaries boundaries = BucketBoundaries.create(
+      Lists.newArrayList(0., 100., 200., 400., 1000., 2000., 4000.));
+  private static final StatsRecorder statsRecorder = Stats.getStatsRecorder();
+
+  public static void main(String[] args) throws IOException, InterruptedException {
+    // Register the view. It is imperative that this step exists,
+    // otherwise recorded metrics will be dropped and never exported.
+    View view = View.create(
+        Name.create("task_latency_distribution_4"),
+        "The distribution of the task latencies.",
+        latencyMs,
+        Aggregation.Distribution.create(boundaries),
+        Collections.emptyList());
+
+    ViewManager viewManager = Stats.getViewManager();
+    viewManager.registerView(view);
+
+    // [START setup_exporter]
+    // Enable OpenCensus exporters to export metrics to Stackdriver Monitoring.
+    // Exporters use Application Default Credentials to authenticate.
+    // See https://developers.google.com/identity/protocols/application-default-credentials
+    // for more details.
+    StackdriverStatsExporter.createAndRegister();
+    // [END setup_exporter]
+
+    // [START example_code]
+    // Record 100 fake latency values between 0 and 5 seconds.
+    long millisPerSecond = 1000;
+    Random rand = new Random();
+    for (int i = 0; i < 100; i++) {
+      long ms = (long) (5 * millisPerSecond * rand.nextDouble());
```

Comment:
Use TimeUnit for more readable time conversions
```suggestion
      long ms = (long) (TimeUnit.MILLISECONDS.convert(5, TimeUnit.SECONDS) * rand.nextDouble());
```

Reply:
Done